curriculum/challenges/english/25-front-end-development/lab-book-inventory-app/66a207974c806a19d6607073.md

```diff
@@ -192,14 +189,14 @@ for (let e of rateSpans) {
 const readBooksRates = document.querySelectorAll('.read .rate');
 assert.isAbove(readBooksRates.length, 0);
 for (let e of readBooksRates) {
-    assert(['one', 'two', 'three'].includes(e.classList[1]));
+    assert.oneOf(e.classList[1], ['one', 'two', 'three']);
 }
 ```
 
 You should have an attribute selector to target rows that have the class of `read`.
 
 ```js
-assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"]'));
+assert.isNotEmpty(new __helpers.CSSHelp(document).getStyle('tr[class="read"]'));
```

Comment:
`getStyle` returns `null` if the selector is not there. `exists` or `isNotNull` are more appropriate

```suggestion
assert.exists(new __helpers.CSSHelp(document).getStyle('tr[class="read"]'));
```